scripts/o3de/o3de/export_project.py

```diff
@@ -377,39 +377,43 @@ def add_args(subparsers) -> None:
 
 def get_asset_processor_batch_path(tools_build_path: pathlib.Path,
                                    using_installer_sdk: bool = False,
+                                   tool_config: str = PREREQUISITE_TOOL_BUILD_CONFIG,
```

Comment:
Can you update the `export_source_built_project.py` calls to these methods to pass in the tool_config option?